tests/Integrations/Nette/V2_4/NetteTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\Nette\V2_4;
+
+use DDTrace\Tag;
+use DDTrace\Tests\Common\SpanAssertion;
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\RequestSpec;
+use DDTrace\Type;
+
+final class NetteTest extends WebFrameworkTestCase
+{
+    protected static function getAppIndexScript()
+    {
+        return __DIR__ . '/../../../Frameworks/Nette/Version_2_4/www/index.php';
+    }
+
+    protected static function getEnvs()
+    {
+        return array_merge(parent::getEnvs(), [
+            'DD_SERVICE' => 'nette_test_app',
+            'DD_TRACE_DEBUG' => true,
```

Comment:
Was this left in from debugging? (Same for the v3 test below.)

Reply:
If I'm not mistaken the `DD_SERVICE` is required so that I can match it to the expected value on the spans, however the `DD_TRACE_DEBUG` was just set for debugging, are there any downsides to keeping it in the test? Other tests also seem to have it.

Reply:
Yes, `DD_SERVICE` is expected, but ideally we'd be testing the integration without `DD_TRACE_DEBUG=1` unless we're testing for a specific debug message (which is pretty common in the phpt tests.) But other than that, if there are other tests that have debug mode turned on, it's most likely accidentally left over from debugging. :)

Reply:
I've removed it.